src/main/java/com/nkh/ECommerceShop/exception/ControllerExceptionHandler.java

```diff
@@ -31,8 +31,8 @@ public ResponseEntity<ErrorMessage> resourceNotFoundException(ResourceNotFoundEx
         return new ResponseEntity<>(message, HttpStatus.NOT_FOUND);
     }
 
-    @ExceptionHandler(AlreadyExistsException.class)
-    public ResponseEntity<ErrorMessage> resourceNotFoundException(AlreadyExistsException ex, WebRequest request) {
+    @ExceptionHandler({AlreadyExistsException.class})
```

Comment:
Why you added the curly brace around the Exception class, that is used for multiple classes?!

Reply:
fixed. just had an issue - played a bit with different solutions and forgot to remove.